src/main/java/org/jabref/gui/EntryFromIDViewModel.java

```diff
@@ -0,0 +1,228 @@
+package org.jabref.gui;
+
+import java.util.Optional;
+
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.FXCollections;
+import javafx.concurrent.Task;
+import javafx.concurrent.Worker;
+
+import java.util.regex.Pattern;
+import java.util.HashMap;
+import  java.util.function.Predicate;
+
+import javafx.scene.control.Label;
+import javafx.scene.control.TextArea;
+import org.jabref.Globals;
+import org.jabref.gui.duplicationFinder.DuplicateResolverDialog;
+import org.jabref.logic.bibtex.DuplicateCheck;
+import org.jabref.logic.bibtexkeypattern.BibtexKeyGenerator;
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.IdBasedFetcher;
+import org.jabref.logic.importer.NoFetcherFoundException;
+import org.jabref.logic.importer.WebFetchers;
+import org.jabref.logic.importer.fetcher.*;
```

Comment:
Your workspace doesn't seem to be fully setup according to https://github.com/JabRef/jabref/wiki/Guidelines-for-setting-up-a-local-workspace

We don't have imports with `*`. Please ensure that you followed **all** steps given there.